xchange-bittrex/src/main/java/org/knowm/xchange/bittrex/dto/trade/BittrexOpenOrder.java

```diff
@@ -2,7 +2,13 @@
 
 import com.fasterxml.jackson.annotation.JsonProperty;
 import java.math.BigDecimal;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.ToString;
 
+@Getter
+@Builder
+@ToString
```

Comment:
Builder
ToString

seam not needed here

we only use the constructor for initialization (actually the Jackson library does) in DTOs
toString is only used for debuging and printing the entire object is no better then just browsing it in the debuggers UI
therefore I think we should avoid it in DTO's all together unless you can think of a sort string representation of the object that would be more helpful then the debuggers UI

Reply:
What's "seam"?

I don't think Builder is used in this example.

ToString is definitely worth having though.  I certainly use it a printed representation in trace and debug statements all over the place.

And, as a principle, lomboking a module should not change anything that previously existed.

Reply:
Yes I added Builder. My frustration came when needing to create this object for unit testing, I had to provide every property to the constructor, when functionally it wasn't required for the test.

It seems that this holds true for this object. Every property is required to construct, when it is just a POJO. This leads to putting in null values. Which should be avoided as well.